R/query-string.R

```diff
@@ -8,29 +8,36 @@ queryStringFilter <- function(req){
 #' @importFrom utils URLdecode
 #' @noRd
 parseQS <- function(qs){
-  if (is.null(qs) || length(qs) == 0 || qs == ""){
+  # Is there data in the request?
```

Comment:
Do you mean we should not allow arbitrary data in the querystring? I've let this sink, and my feeling is that it perhaps it makes sense prescribe the use of POST for including JSON data in the body, and only accept querystrings in the GET request. 

Perhaps that is what you mean by suggesting to move this functionality to a new function called parseBody. If I understand correctly the higher function would need to check the kind of request (GET or POST) and pass the check for a query string in case of a GET request, and check for a body in case of POST. It does [not seem to make sense to post data with a GET request](http://stackoverflow.com/questions/978061/http-get-with-request-body).